### DISCUSSION TITLE
Multiline card title

### DISCUSSION THREAD

**Discussion by diedder** [Budibase/budibase#6291] [Help] [answered]
Hi,

is there a way to display a long text with multiple lines as card title? I want to feed the card title with a fomatted, mutli line text column of a table.

Thanks for help!

diedder

**Comment by melohagan** [comment, marked as answer]
Hey @diedder 

You would need to use global styling to accomplish this: https://docs.budibase.com/docs/custom-css#global-styling

Here is the default CSS:
```
.spectrum-Card-title {
    white-space: nowrap;
    overflow: hidden;
    text-overflow: ellipsis;
}
```

So you could replace the `white-space` with `white-space: normal !important;` in your embed.

**Reply by diedder** [reply]
Hi @melohagan,

thank you again for your quick support!

unfortunately I have no CSS skills... Can you also help to fix this overlap?
![image](https://user-images.githubusercontent.com/25912001/173347601-cd965a77-308d-430c-8233-40ee4cb04534.png)

diedder

**Reply by melohagan** [reply]
You can add some `margin-top` to the `.spectrum-Card-footer` class. It contains the button.

I'm no CSS guru myself, but a handy tip is to right-click inspect the element. Then you can directly fiddle with the CSS values in the Elements tab, to see what you need:

![Screenshot 2022-06-13 at 13 01 52](https://user-images.githubusercontent.com/101575380/173349256-3b599a21-576b-44c2-ac71-968798320d79.png)

**Reply by diedder** [reply]
```
.spectrum-Button {
    margin-top:20px;
}
```
works fine!

**Reply by melohagan** [reply]
Awesome - just be aware that'll affect all the buttons on your page, not just the one in the card title. That's why I suggested the `.spectrum-Card-footer` but either way global styling should be used infrequently and with caution!

**Reply by diedder** [reply]
I was to fast with my anser....
Multiline  need more than just the wrap. I tried some longer text. The text is growing over the button and even out of the whole card. I think I need some CSS which will resize the whole card according to the height of the `spectrum-card-title` div
![image](https://user-images.githubusercontent.com/25912001/173354096-3cb6c495-f8e3-4c18-b68f-1fc3281cfe76.png)

some CSS-guru here?

**Reply by melohagan** [reply]
Another thought as well, is if using a Card is the appropriate component in this case? Maybe you would get more customization with a container? Then you could add in paragraphs, headers, etc.

**Reply by diedder** [reply]
great thougt! I reworked it an built my own cards with containers. It#s easier an much more flexbible. Thank You!
